opendebates/tasks.py

```diff
@@ -63,7 +63,7 @@ def update_recent_events():
                 "voter__user",
                 "submission__category",
             ).filter(
-                submission__category__site_mode_id=site_mode.id,
+                submission__category__debate=debate,
```

Comment:
It turned out that Django does the smart thing, and does the direct comparison of ids anyway, instead of joining to the debate table.